csrc/dynamic_type.h

```diff
@@ -124,21 +138,39 @@ namespace nvfuser {
 #pragma GCC diagnostic ignored "-Wbool-operation"
 #endif
 
-template <typename... Ts>
-struct DynamicType;
+template <template <typename> typename... Templates>
```

Comment:
`Templates` is a template, not a type. Just like `std::vector` is a template, `std::vector<int>` is a type.